nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -73,10 +77,8 @@ def wrapped(*args, **kwargs):
         ctx.in_operator = True
 
         try:
-            if operator_info.custom_trace_fn is not None:
-                result = operator_info.custom_trace_fn(operator, *args, **kwargs)
-            elif ctx.is_forwarding:
-                from nncf.torch.dynamic_graph.patch_pytorch import ForwardTraceOnly
+            if ctx.is_forwarding:
+                from nncf.torch.dynamic_graph.trace_functions import ForwardTraceOnly
```

Comment:
If we at this moment never instantiate `ForwardTraceOnly` objects, you might as well refactor it so that it becomes a single function rather than a callable instance of a single-function class.

Reply:
Done